exercises/connect/connect_test.rb

```diff
@@ -1,112 +0,0 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
-require 'minitest/autorun'
-require_relative 'connect'
-
-# Test data version:
-# 8268727
-class ConnectTest < Minitest::Test
```

Comment:
Even though it's generated, the generated test file still needs to be present in the repository.